packages/sdk/src/across/gasFeeCalculator.ts

```diff
@@ -0,0 +1,40 @@
+import { BigNumber } from "ethers";
+import { ConvertDecimals } from "../utils";
+import { toBNWei, BigNumberish } from "./utils";
+
+export const SLOW_ETH_GAS = 243177;
+export const SLOW_ERC_GAS = 250939;
+export const SLOW_UMA_GAS = 273955;
+
+// fast costs are slightly higher and include the slow cost
```

Comment:
why hard code these? could you not use the [ethers estimateGas](https://docs.ethers.io/v5/api/contract/contract/#contract-estimateGas) feature to make this file more future proof? For instance, whenever Ethereum hard-forks, function gas costs will change as intruction costs change, meaning that these gas limits will not always be accurate.